src/lightning_app/runners/cloud.py

```diff
@@ -481,23 +494,26 @@ def _ensure_cluster_project_binding(self, project_id: str, cluster_id: str):
     def _check_uploaded_folder(root: Path, repo: LocalSourceCodeDir) -> None:
         """This method is used to inform the users if their folder files are large and how to filter them."""
         lightning_tar = set(fnmatch.filter(repo.files, "*lightning-*.tar.gz"))
-        app_folder_size = sum(Path(p).stat().st_size for p in repo.files if p not in lightning_tar)
-        app_folder_size_in_mb = round(app_folder_size / (1000 * 1000), 5)
+        files = [Path(f) for f in repo.files]
+        file_sizes = {f: f.stat().st_size for f in files}
+        mb = 1000 * 1000
+        app_folder_size_in_mb = sum(v for k, v in file_sizes.items() if k not in lightning_tar) / mb
         if app_folder_size_in_mb > CLOUD_UPLOAD_WARNING:
-            path_sizes = [(p, Path(p).stat().st_size / (1000 * 1000)) for p in repo.files]
-            largest_paths = sorted((x for x in path_sizes if x[-1] > 0.01), key=lambda x: x[1], reverse=True)[:25]
```

Comment:
I removed the filter for files that are less than 0.01 MB. I think it's still worth showing them to the user.

If you disagree, I can also add it back. We just need to avoid showing an empty "Here are the largest files: ..." message